Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM golang:1.14.3-alpine3.11 as build
+
+ENV GOPATH=/go
+RUN mkdir -p $GOPATH/src/github.com/joshprzybyszewski/cribbage
+WORKDIR $GOPATH/src/github.com/joshprzybyszewski/cribbage
```

Comment:
this `RUN` isn't needed. [link](https://docs.docker.com/engine/reference/builder/#workdir)